kratos/python/add_dofs_to_python.cpp

```diff
@@ -44,6 +44,12 @@ void  AddDofsToPython(pybind11::module& m)
     .def("Fix", &Dof<double>::FixDof)
     .def("Free", &Dof<double>::FreeDof)
     .def("IsFixed", &Dof<double>::IsFixed)
+    .def("__lt__", [](Dof<double>& dof1, Dof<double>& dof2){return dof1<dof2; })
```

Comment:
this is just to allow sorting dofs in python, a feature i will need for the next step i am planning

Reply:
sorting dofs is definitely possible, we are doing this all the time in the strategy to Unique the dof list.

i added this since i plan to provide an interface to provide dofs, that will require me to be able to compare dofs, and eventually to sort lists of dofs. If you see i am doing nothing else than providing the comparison that is implemented in c++